taskpools/chase_lev_deques.nim

```diff
@@ -111,13 +109,21 @@ proc grow[T](deque: var ChaseLevDeque[T], buf: var ptr Buf[T], top, bottom: int)
   for i in top ..< bottom:
     tmp[][i] = buf[][i]
 
-  buf.prev = deque.garbage
+  buf.prevRetired = deque.garbage
   deque.garbage = buf
   # publish globally
   deque.buf.store(tmp, moRelaxed)
   # publish locally
   swap(buf, tmp)
 
+proc garbageCollect(deque: var ChaseLevDeque) {.inline.} =
```

Comment:
In `grow`, there is a comment referring to `garbageUsed`, but that `garbageUsed` doesn't seem to exist. Is there something missing, or should the comment be cleaned up?